### ISSUE TITLE
error on boot

### ISSUE THREAD

**Issue by fullmetal1** [XNBlank/discord-linux#17]
I get this error on boot. The only solution seems to be to ininstall and reinstall the .deb package

I'm on Linux Mint Cinnamon 17.3 64 bit

```
A JavaScript error occurred in the main process
Uncaught Exception:
TypeError: Cannot read property 'minTray' of undefined
    at Object.<anonymous> (/usr/local/bin/discord-linux-x64/resources/app/main.js:22:17)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Object.<anonymous> (/usr/local/bin/discord-linux-x64/resources/atom.asar/browser/lib/init.js:123:10)
    at Object.<anonymous> (/usr/local/bin/discord-linux-x64/resources/atom.asar/browser/lib/init.js:125:4)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
```

**Comment by jacobmischka** [comment]
Yours is breaking because there is supposed to be an `init.json` file at `[install dir]/resources/app/init.json`, it looks like it can't find it for some reason. Probably because the application didn't close properly, or because of incorrect permissions.

This is a problem because that directory is usually somewhere owned by root, and we don't really want a simple configuration json somewhere that's owned by root.

This should be moved to `~/.config/Discord/` whenever @XNBlank or I find the time.

**Comment by jacobmischka** [comment]
Should be fixed with #18.

**Comment by XNBlank** [comment]
This has been fixed.
